src/rpm/package.rs

```diff
@@ -375,7 +375,7 @@ impl PackageMetadata {
         })
     }
 
-    pub(crate) fn write(&self, out: &mut impl io::Write) -> Result<(), Error> {
+    pub fn write(&self, out: &mut impl io::Write) -> Result<(), Error> {
```

Comment:
Please write a docstring

Reply:
I wasn't sure if I should use `RPM header` or `PackageMetadata` but landed on `RPM header`.

`/// Write the RPM header to a buffer`

Let me know if I should change that.

Reply:
It's fine